README.md

```diff
@@ -65,17 +65,18 @@ If you followed the previous steps you should already be inside your local clone
 2. Open Unity and inside the project launcher select the ![Open](doc/source/images/unity_open.png?raw=true) button.
 3. Navigate to where you cloned this repository and open the "SpeechSandbox" directory.
 4. If prompted to upgrade the project to a newer Unity version, do so.
-5. Follow [these instructions](https://github.com/IBM/unity-sdk#getting-the-watson-sdk-and-adding-it-to-unity) to add the Watson Unity SDK downloaded in step 1 to the project.
-6. Follow [these instructions](https://github.com/IBM/unity-sdk#configuring-your-service-credentials) to add your Speech To Text and Conversation service credentials (located on [IBM Bluemix](https://console.ng.bluemix.net/)).
-7. Select `Advanced Mode` in the configuration window.
-8. Open the script vr-speech-sandbox-cardboard/SpeechSandbox/Assests/Scripts/VoiceSpawner.cs and put your workspace ID on line #34 in the Start() method.
+5. You will need to change the Build Settings to Android in order to bring up the configuration tab for the Watson unity-sdk. Go to File -> Build Settings (Ctrl + Shift +B) and change the Platform to `Android`, then click `Switch Platform`.
+6. Follow [these instructions](https://github.com/IBM/unity-sdk#getting-the-watson-sdk-and-adding-it-to-unity) to add the Watson Unity SDK downloaded in step 1 to the project.
+7. Follow [these instructions](https://github.com/IBM/unity-sdk#configuring-your-service-credentials) to add your Speech To Text and Conversation service credentials (located on [IBM Bluemix](https://console.ng.bluemix.net/)).
+8. Select `Advanced Mode` in the configuration window.
+9. Open the script vr-speech-sandbox-cardboard/SpeechSandbox/Assests/Scripts/VoiceSpawner.cs and put your workspace ID on line #34 in the Start() method.
  You can find your workspace ID by selecting the expansion menu on your conversation workspace and selecting `View details`.
     ![View Details Location](doc/source/images/workspace_details.png?raw=true)
-9. In the Unity editor project tab, select Assets->Scenes->MainGame->MainMenu and double click to load the scene.
-10. Press Play
-11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Ctrl + Shift +B) and click Build. 
-12. When prompted you can name your build and then move it to your phone.
-13. Alternately, connect the phone via USB and File-> Build and Run (or Ctrl+B).
+10. In the Unity editor project tab, select Assets->Scenes->MainGame->MainMenu and double click to load the scene.
```

Comment:
Assets->Scenes->MainGame->MainMenu should also be wrapped in underscores:

 _Assets_ -> _Scenes_ -> _MainGame_ -> _MainMenu_

Reply:
cool. I actually need to change that to Assets->Scenes->Playground for this version.